dashboard/config/locales/en.yml

```diff
@@ -1385,4 +1385,19 @@ en:
   course_deprecated:
     title: "Heads up! This curriculum is no longer available, but Code.org has newer options available for you."
     button: "View new curriculum"
-
+  hoc_tutorial_email_subject: "Rock the Hour of Code with Code.org Tutorials!"
+  hoc_tutorial_email_content:
+    dear_teacher: "Dear %{teacher_name},"
+    intro_step_by_step_guide: "We're so excited that you're holding an Hour of Code event using Code.org's free activities and tutorials! We're honored to have educators like you introducing students to the exciting world of coding. Here's a step-by-step guide to ensure a successful Hour of Code event:"
+    sign_up_step: "**Sign Up** (Optional… but awesome!): Create a free Code.org account for access to a personal dashboard where you can manage your classroom's Hour of Code! [Sign up here](%{url})."
+    create_classroom_section: "**Create a Classroom Section** (Also optional… but super useful!): Creating a classroom section allows you to track your students' progress, view their work, and even leave them feedback! [Set up your section here](%{section_setup_url}). And for all our newbies, here's a [handy guide](%{guide_url}) to help you out!"
+    pick_perfect_activity: "**Pick the Perfect Activity**: Select the best Hour of Code tutorial for your event. Explore the options and assign your chosen tutorial to your section, so your students will know exactly where to start."
```

Comment:
can we link "Select the best Hour of Code tutorial" to https://studio.code.org/catalog?marketingInitiative=hoc